geomstats/geometry/beta_distributions.py

```diff
@@ -19,6 +19,13 @@ class BetaDistributions(EmbeddedManifold):
     r"""Class for the manifold of beta distributions.
 
     This is :math: Beta = `R_+^* \times R_+^*`.
+
+    Attributes
+    ----------
+    dim : int
+        Dimension of the manifold of beta distributions.
+    embedding_manifold : Manifold
+        Embedding manifold.
```

Comment:
Actually, as per your response in an earlier PR, as the beta manifold is an open set of R^2 it should not be considered an embedded manifold of R^2

Reply:
Hm, let's re-discuss this? In this case, I would say that the quadrant [0, \infinity]^2 can be seen as an embedded manifold in R^2.